skipper/runner.py

```diff
@@ -104,10 +111,12 @@ def handle_volumes_bind_mount(docker_cmd, homedir, volumes, workspace):
     else:
         volumes.extend([
             '%(workspace)s:%(workspace)s:rw,Z' % dict(workspace=workspace),
-            '/var/lib/osmosis:/var/lib/osmosis:rw,Z',
             '/var/run/docker.sock:/var/run/docker.sock:Z',
             '%s:/opt/skipper/skipper-entrypoint.sh:Z' % utils.get_extra_file("skipper-entrypoint.sh"),
             ])
+        # Will fail on Mac
+        if os.path.exists('/var/lib/osmosis'):
```

Comment:
lets move it from if else, looks like it is the same in podman and docker

Reply:
@tsorya but it is different with podman we don't use :Z
Don't know enough about podman but with docker two concurrent skipper run will fail